packages/design-system/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -288,6 +289,16 @@ export const Autocomplete = (props: AutocompleteProps) => {
     },
   };
 
+  const oldItems = usePrevious(items);
+  useEffect(() => {
+    // If the items come in significantly later than when the user started typing,
+    // react-stately will not realize that it should be showing those results. There
+    // might be items, but `isOpen` will be false 🤦‍♂️.
+    if (state.isFocused && items && items !== oldItems && items.length !== oldItems?.length) {
```

Comment:
Realizing that `items !== oldItems && items.length !== oldItems?.length` is redundant, and what I meant was `(items !== oldItems || items.length !== oldItems?.length)` for cases where they might be passing the same array instance...in which case I guess it would also fail the length check. I'm just going to remove the length check. Users can't expect to pass in the same instance of an array to any React component and have it cause an update.